addons/test_mail/tests/test_mail_message.py

```diff
@@ -252,6 +252,67 @@ def test_mail_message_values_fromto_reply_to_force_new(self):
         self.assertNotIn('-%d-' % self.alias_record.id, msg.message_id.split('@')[0])
 
 
+    def test_mail_message_starred_count(self):
+        partner = self.partner_employee
+        user = self.user_employee
+        ticket = self.env['mail.test.ticket'].with_user(user).with_context(self._test_context).create({
+            'name': 'Test',
+        })
+        message = ticket.message_post(body="TestingMessage", message_type="comment")
+
+        ## Body case
```

Comment:
Actually I think I would write test on ``is_empty`` (and as a side effect check star status). But I think the important stuff to test here is the is_empty itself.